README.md

```diff
@@ -111,12 +113,25 @@ This is a schematic logging configuration to show log messages from input_nameA
 - `ovirt` type - `ovirt` input supports oVirt specific inputs.<br>
    For the details, visit [oVirt Support](../../design_docs/rsyslog_ovirt_support.md).
 
+- `relp` type - `relp` input supports receiving logs from the remote logging system over the network using relp. This input type makes rsyslog a server.<br>
```

Comment:
Do we need the part about "This input type makes rsyslog a server."?

Reply:
Let me remove the sentence. Remote input also has it. Shall I remove it, as well?
`129 - `remote` type - `remote` input supports receiving logs from the remote logging system o    ver the network. This input type makes rsyslog a server.<br>`

Reply:
yes - I think it might be confusing to say "makes rsyslog a server" if someone can misinterpret that to ask "Does that mean it doesn't act as a client anymore?"